lib/elixir/pages/anti-patterns/code-anti-patterns.md

```diff
@@ -4,7 +4,42 @@ This document outlines anti-patterns related to your code and particular Elixir
 
 ## Comments
 
-TODO.
+#### Problem
+
+When you overuse comments or comment self-explanatory code, it can have the effect of making code *less readable*.
```

Comment:
I agree with this example and circumstance, where comments are made unnecessarily to compensate for poorly 'declarative' code. However, in the context of Elixir, I understand that this smell also applies in situations where it is necessary to document a function, but it is done through comments rather than using @doc.

Reply:
> @lucasvegi those are completely different concerns IMO. I think that's about documentation, not about comments.

But you agree with me that this is not a good practice, okay (documentation by comments)?

Reply:
@whatyouhide I think we should add a note that Elixir make a clear distinction between code comments and documentation and that they have different audiences. We can refer them to the "writing-documentation.md" guide.

Reply:
@josevalim done.

> But you agree with me that this is not a good practice (documentation by comments), ok?

@lucasvegi public documentation through comments is not a good practice, no. Private documentation through comments is totally fine, IMO.